### ISSUE TITLE
Error: Symbol's value as variable is void: rtm-contacts-add

### ISSUE THREAD

**Issue by dotemacs** [mbunkus/simple-rtm#10]
Hello,

I'd like to report a potential bug and a quick & dirty workaround.

Installed `simple-rtm` via [MELPA](http://melpa.org/#/), which installed: simple-rtm-20140926.439 (which is as expected, as the current last commit in the master is: mbunkus/simple-rtm@cff3e87bc2d440aa643c9f43bf99972668147231).

Then I tried to execute: <kbd>M-x simple-rtm-mode RET</kbd>, but when I do, I get this error:

    Symbol's value as variable is void: rtm-contacts-add

:confused:

I then commented [out this section in rtm.el](https://github.com/mbunkus/simple-rtm/blob/cff3e87bc2d440aa643c9f43bf99972668147231/lisp/rtm.el#L230-L233) and then: <kbd>M-x simple-rtm-mode</kbd> at which point I get prompted to go to the URL to authenticate the app. 

Once I authenticate I can then run <kbd>M-x simple-rtm-mode RET</kbd> to load and browse the tasks. :tada:

:exclamation: My <kbd>M-x emacs-version RET</kbd>:

    GNU Emacs 24.4.1 (x86_64-apple-darwin13.4.0, NS apple-appkit-1265.21)

**Comment by hakuch** [comment]
I had the same exact issue (also on OS X), and the fix worked after I manually re-compiled the simple-rtm-mode ELPA directory.

My version string is:

```
GNU Emacs 24.4.1 (x86_64-apple-darwin14.0.0, NS apple-appkit-1343.16)
```

**Comment by feoh** [comment]
+1 - same issue, more recent emacs version: This is GNU Emacs 25.0.50.1 (x86_64-apple-darwin14.3.0, NS appkit-1347.57 Version 10.10.3 (Build 14D136))
 of 2015-05-31 on MBP15-04484.local

**Comment by feoh** [comment]
Thanks @hakuch that byte compile fix worked for me too. @mbunkus Seems like you might be too busy to maintain this which is fine - could you maybe please throw us a bone and give us a hint here? :) Is maybe something just missing from the autoload definitions?

**Comment by mbunkus** [comment]
I can only repeat what I've said above: I'm not the original author of `rtm.el`, and I don't understand its intricacies at all. I've only included it in my repo as it doesn't seem to be released on its own (never mind that it hadn't been updated in quite some time back when I started using it). I cannot really help with this issue, nor can I guess what the issue might be.

I'd be happy to include a fix anyone else comes up with, of course – my intention is surely not to leave it broken in perpetuity. On the other hand I'm not willing to spend time debugging this either, especially as I've never been able to reproduce it. Sorry.
